From c24676272f853bc6f566bae4898a09953fc69a06 Mon Sep 17 00:00:00 2001
From: Julien Gros <julien.gros@intel.com>
Date: Tue, 8 Jan 2013 15:46:30 +0100
Subject: [PATCH 11/12] [BT] TI WLxx integration in MR1

BZ: 72228

Multi-vendor support for bt_vendor configuration

Change-Id: I6ee8fca5ec7033388e79ff447418386344a8453a
Signed-off-by: Julien Gros <julien.gros@intel.com>
Reviewed-on: http://android.intel.com:8080/85605
Reviewed-by: Bondier, CedricX <cedricx.bondier@intel.com>
Reviewed-by: Aquilone, RaffaeleX <raffaelex.aquilone@intel.com>
Reviewed-by: Biselli, MirkoX <mirkox.biselli@intel.com>
Reviewed-by: Chen, HuaqiangX <huaqiangx.chen@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 libbt/conf/intel/common/bt_vendor.conf     |    5 -----
 libbt/conf/intel/common/bt_vendor_bcm.conf |    5 +++++
 libbt/conf/intel/common/bt_vendor_ti.conf  |    5 +++++
 3 files changed, 10 insertions(+), 5 deletions(-)
 delete mode 100644 libbt/conf/intel/common/bt_vendor.conf
 create mode 100644 libbt/conf/intel/common/bt_vendor_bcm.conf
 create mode 100644 libbt/conf/intel/common/bt_vendor_ti.conf

diff --git a/libbt/conf/intel/common/bt_vendor.conf b/libbt/conf/intel/common/bt_vendor.conf
deleted file mode 100644
index 62ffeb9..0000000
--- a/libbt/conf/intel/common/bt_vendor.conf
+++ /dev/null
@@ -1,5 +0,0 @@
-# UART device port where Bluetooth controller is attached
-UartPort = /dev/ttyMFD0
-
-# Firmware patch file location
-FwPatchFilePath = /etc/firmware/
diff --git a/libbt/conf/intel/common/bt_vendor_bcm.conf b/libbt/conf/intel/common/bt_vendor_bcm.conf
new file mode 100644
index 0000000..62ffeb9
--- /dev/null
+++ b/libbt/conf/intel/common/bt_vendor_bcm.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFilePath = /etc/firmware/
diff --git a/libbt/conf/intel/common/bt_vendor_ti.conf b/libbt/conf/intel/common/bt_vendor_ti.conf
new file mode 100644
index 0000000..26a7e5f
--- /dev/null
+++ b/libbt/conf/intel/common/bt_vendor_ti.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/hci_tty
+
+# Firmware patch file location
+FwPatchFilePath = /etc/firmware/
-- 
1.7.9

